components/docs-chef-io/content/automate/ha_upgrade_introduction.md

```diff
@@ -12,4 +12,40 @@ gh_repo = "automate"
     weight = 200
 +++
 
-TO BE ADDED
\ No newline at end of file
+## What is Upgradation?
+
+Upgradation is a process of replacing older version with improved newer version with some new features or/and fixes for bugs that has been noticed in older versions. Use the --major flag to upgrade from the latest to a major version. This section will talk about the version upgrade of Chef Automate with its possible scenarios.
```

Comment:
REPLACE IT WITH: **Upgradation** replaces an older version with an improved newer version with some new features or/and fixes for bugs that were present in older versions. Use the --major flag to upgrade from the latest to a major version. This section will talk about the version upgrade of Chef Automate with its possible scenarios.

Reply:
Done